src/common/MatrixHandler.h

```diff
@@ -146,12 +146,12 @@ public :
 
 				double min = *std::min_element(distances.begin(), distances.end());
 
-                map<double, pair< std::pair<double, double>,  pair<int, int> > >::iterator near = helper.find(min);
```

Comment:
Why did you change the variable : just for readability or there is something more we need to know ?

Reply:
It's because windef.h (which is included by windows.h, which we include in win_time.h) defines macros for 'near' and 'far' so we can't use those as variable names. I've left the info in the [commit](https://github.com/ecmwf/magics/pull/3/commits/5e26938f83d1281c184abf6467aeb16917d35e3a) message for future reference.

Reply:
Sorry, I had not read the commit message .. 
Good to know too ..